kubernetes/cmsweb/scripts/deploy-secrets.sh

```diff
@@ -2,16 +2,32 @@
 # helper script to deploy given service with given tag to k8s infrastructure
 
 if [ $# -ne 3 ]; then
-    echo "Usage: deploy-secrets.sh <namespace> <service-name> <path_to_configuration> <reference_of_decryption_barbican_secret>"
+    echo "Usage: deploy-secrets.sh <namespace> <service-name> <path_to_configuration>"
     exit 1
 fi
 
+cluster_name=`kubectl config get-clusters | grep -v NAME`
+
 ns=$1
 srv=$2
 conf=$3
-secretref=${4:-https://openstack.cern.ch:9311/v1/secrets/35145b52-18af-47a3-90d0-1861c51a9c65}
+tmpDir=/tmp/$USER/sops
+if [ -d $tmpDir ]; then
+   rm -rf $tmpDir
+fi
+mkdir -p $tmpDir
+cd $tmpDir
+
 
+if [ -z "`command -v sops`" ]; then
+  # download soap in tmp area
+  wget -O sops https://github.com/mozilla/sops/releases/download/v3.7.2/sops-v3.7.2.linux.amd64
```

Comment:
Since we download new script in user's home/bin area I suggest to put appropriate message here, e.g.
`echo "Download and install sops scripts under $HOME/bin"`